src/main/java/com/github/krzysiekjodlowski/ox/ConsoleUI.java

```diff
@@ -0,0 +1,32 @@
+package com.github.krzysiekjodlowski.ox;
+
+import java.util.Scanner;
+
+/**
+ * Interacting directly with user,
+ * used for collecting input end
+ * outputting messages to user
+ *
+ * @author Krzysztof Jodlowski
+ */
+class ConsoleUI implements UI<String, Integer> {
+    private final Scanner scanner;
+
+    ConsoleUI(Scanner scanner) {
+        this.scanner = scanner;
+    }
+
+    @Override
+    public void say(String message) {
+        System.out.println(message);
```

Comment:
Think about injecting Print Stream in constructor instead of hard coding system.out

Reply:
I like that, done:)